src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -110,12 +111,15 @@ class GroupTagValues extends AsyncComponent<
       <TableWrapper>
         <h3>
           {tag.key === 'user' ? t('Affected Users') : tag.name}
-          <a
+          <Button
+            size="small"
+            priority="default"
             href={`/${orgId}/${group.project.slug}/issues/${group.id}/tags/${tagKey}/export/`}
-            className="btn btn-default btn-sm m-left m-right"
+            className="m-left m-right"
```

Comment:
instead of adding a className to the Button could we do a styled component? 

```javascript
const ExportButton = styled('Button')`
    margin: 0 ${space(1.5)};
`;
```

and then remove from the `TableWrapper` component.

Reply:
Changed it :D